docs/pipelines.md

```diff
@@ -319,21 +285,17 @@ For example see this `Pipeline` spec:
         resource: my-repo
 ```
 
-In this `Pipeline`, we want to test the code before we build from it, but there
-is no output from `test-app`, so `build-app` uses `runAfter` to indicate that
-`test-app` should run before it, regardless of the order they appear in the
-spec.
+### Using the `retries` parameter
 
-#### retries
+For each `Task` in the `Pipeline` you can specify the number of times its
+execution should be retried if it fails. Any issues the `Task` is expected
+to encounter, such as network connectivity or missing dependencies, must
+return `False` for the "TaskRun Status Succeeded" `Condition`. If you don't
```

Comment:
Suggest rewording the sentence about TaskRun Status Succeeded a little bit:

> When a Pipeline Task ends in failure, its `TaskRun` will have a `ConditionSucceeded` with value `False`.
The `retries` parameter allows the `Task` to be reattempted when this happens. This can be useful to set when you know there are likely to be issues with network connectivity or missing dependencies during Task execution.

Reply:
Sorry, should be "`Succeeded` Condition" instead of "`ConditionSucceeded`".

Reply:
Reworded.